pkgs/development/python-modules/zest_releaser/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonPackage
+, colorama
+, fetchFromGitHub
+, lib
+, requests
+, setuptools
+, six
+, twine
+, test_setup
```

Comment:
How is this packaged used in zest_releaser? I can't find any usage in the repo but I did not dig very deep. I think our best option would be to try to remove it if it is easily possible as it seems to be very dead.